config/crd/bases/match.gatekeeper.sh_matchcrd.yaml

```diff
@@ -0,0 +1,209 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.10.0
+  creationTimestamp: null
+  name: matchcrd.match.gatekeeper.sh
+spec:
+  group: match.gatekeeper.sh
+  names:
+    kind: DummyCRD
+    listKind: DummyCRDList
+    plural: matchcrd
+    singular: dummycrd
+  scope: Namespaced
+  versions:
+  - name: match
+    schema:
+      openAPIV3Schema:
+        description: DummyCRD is a "dummy" CRD to hold the Match object, which we
+          ultimately need to generate JSONSchemaProps. The TypeMeta and ObjectMeta
+          fields are required for controller-gen to generate the CRD.
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          embeddedMatch:
+            description: Match selects objects to apply mutations to.
```

Comment:
We should change "mutations", since this will also be for constraints now.

"Match selects which objects are in scope"?

Reply:
Does not look like this has been updated?

Reply:
Good catch, somehow I did not upload the commit properly. The latest version should have all of Max's suggestions.